gobrew_test.go

```diff
@@ -25,6 +25,17 @@ func setupGobrew(t *testing.T, ts *httptest.Server) GoBrew {
 	return gb
 }
 
+func BenchmarkInstallGo(t *testing.B) {
+	for i := 0; i < t.N; i++ {
+		ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
```

Comment:
why do you include the launch of the web server in the performance test?
in my opinion, this should not be done in a loop, but at the very beginning of the function, after which we should reset the time counter.
in this case, the data will be correct.

Reply:
Ah. Thanks!!